uber/site_sections/registration.py

```diff
@@ -30,7 +30,7 @@ def check_everything(attendee):
 
 @all_renderable(PEOPLE, REG_AT_CON)
 class Root:
```

Comment:
The registration page is really slow when loading 100 people, but 99% of the time, we don't need to see 100 random people, we just want to do a search.  So I changed it to show 0 people when the page first loads, and let you pick a page if you want.